charts/seed-controlplane/charts/packet-cloud-controller-manager/templates/packet-ccm-deployment.yaml

```diff
@@ -0,0 +1,71 @@
+---
+apiVersion: {{ include "deploymentversion" . }}
+kind: Deployment
+metadata:
+  name: packet-cloud-controller-manager
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: packet-cloud-controller-manager
+    garden.sapcloud.io/role: controlplane
+spec:
+  replicas: {{ .Values.replicas }}
+  selector:
+    matchLabels:
+      app: packet-cloud-controller-manager
+  template:
+    metadata:
+      labels:
+        app: packet-cloud-controller-manager
+{{- if .Values.podAnnotations }}
+      annotations:
+{{ toYaml .Values.podAnnotations | indent 8 }}
+{{- end }}
+    spec:
+      dnsPolicy: Default
+      hostNetwork: true
+      tolerations:
+        # this taint is set by all kubelets running `--cloud-provider=external`
+        # so we should tolerate it to schedule the packet ccm
+        - key: "node.cloudprovider.kubernetes.io/uninitialized"
```

Comment:
OK, removed.

Reply:
This toleration is only needed if you have regular master setup - we don't have this (we deploy control plane to seed).